### DISCUSSION TITLE
Skip the next map in a multiplayer lobby

### DISCUSSION THREAD

**Discussion by apollo-dw** [ppy/osu#18825] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/83023433/175386084-8b26e1c1-252c-41bb-baab-eb98d1a4975b.png)

i was playing round robin - for some reason you can't skip the map that's next up but you can remove all the other ones afterwards which certainly seems strange

**Comment by Theighlin** [comment, marked as answer]
Same as #17534. Was done the current way due to [implementation complexity](https://github.com/ppy/osu-server-spectator/pull/93#discussion_r765734184.)i believe. 
Maybe a proper issue should be opened for tracking this.

**Reply by smoogipoo** [reply]
https://github.com/ppy/osu/issues/18828
